### PR TITLE
boost-ext-ut: Minor updates

### PR DESCRIPTION
**Pull request by jwillikers**
Specify library name and version:  **boost-ext-ut/***

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/boost-ext-ut/all/test_v1_package/CMakeLists.txt [hunk 1]
```diff
@@ -1,14 +1,11 @@
-cmake_minimum_required(VERSION 3.15)
-project(test_package)
-
-set(CMAKE_CXX_STANDARD 20)
-set(CMAKE_CXX_STANDARD_REQUIRED yes)
-set(CMAKE_CXX_EXTENSIONS no)
+cmake_minimum_required(VERSION 3.8)
```
Comment (uilianries) [inline recipes/boost-ext-ut/all/test_v1_package/CMakeLists.txt:1]:
```suggestion
cmake_minimum_required(VERSION 3.12)
```
cxx_std_20 was added only on 3.12

https://cmake.org/cmake/help/latest/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html

File: recipes/boost-ext-ut/all/test_package/CMakeLists.txt [hunk 2]
```diff
@@ -1,11 +1,8 @@
-cmake_minimum_required(VERSION 3.16)
-project(test_package)
-
-set(CMAKE_CXX_STANDARD 20)
-set(CMAKE_CXX_STANDARD_REQUIRED yes)
-set(CMAKE_CXX_EXTENSIONS no)
+cmake_minimum_required(VERSION 3.8)
```
Comment (uilianries) [inline recipes/boost-ext-ut/all/test_package/CMakeLists.txt:1]:
```suggestion
cmake_minimum_required(VERSION 3.12)
```

File: recipes/boost-ext-ut/all/test_v1_package/CMakeLists.txt (hunk 1 above)
Comment (jwillikers) [reply recipes/boost-ext-ut/all/test_v1_package/CMakeLists.txt:1]: Thanks!

**Review by uilianries — APPROVED** [review]
LGTM

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**